assets/products/product_clients/chromebook.py

```diff
@@ -17,14 +17,15 @@ class HPTouchScreenChromebook(BaseProduct):
         walmart_product_image_alt,
     )
     product_type = "Computer"
+    msrp="599"
```

Comment:
Is is possible for you to put a comment here saying essentially where you got this value from / how we came up with it?

Reply:
Yes, I can add this.